example/helm/readme.md

```diff
@@ -20,11 +19,16 @@ Create a cluster
 k3d cluster create tempo --api-port 6443 --port "16686:80@loadbalancer"
 ```
 
+If you wish to use a local image, you can import these into k3d
+
+```console
+k3d image import grafana/tempo:latest --cluster tempo
+```
+
 Next either deploy the microservices or the single binary.
 
 ### Microservices
-The microservices deploy of Tempo is fault tolerant, high volume, independently scalable.  This jsonnet is in use by
```

Comment:
I like this sentence, any particular reason behind removing it? :)

Reply:
This is about the helm config, we don't use this internally. I think it's a copy-paste error.